controller/ingress_entry.go

```diff
@@ -28,8 +28,8 @@ type LoadBalancerEntry struct {
 }
 
 // FilterInvalidEntries returns a slice of all the valid LoadBalancer entries
```

Comment:
For now it's applicable for all of them, although I think it may make more sense to put this logic inside the generic controller rather than standalone on the entry, since it is specific to the controller's logic.